message_ix_models/model/water/data/water_for_ppl.py

```diff
@@ -171,6 +177,91 @@ def hist_cap(x: pd.Series, context: "Context", hold_cost: pd.DataFrame) -> list:
     ]
 
 
+def relax_growth_constraint(ref_hist, scen, cooling_df, g_lo, constraint_type):
+    """
+    Function to check if the parent technologies are shut down and require
+    relaxing the growth constraint.
+
+    Parameters:
+    ref_hist (pd.DataFrame): Historical data.
+    scen (Scenario): Scenario object to retrieve parameters.
+    cooling_df (pd.DataFrame): DataFrame containing cooling technologies and
+    their parent technologies.
+    g_lo (pd.DataFrame): DataFrame containing growth constraints.
+    constraint_type (str): Type of constraint to check ("activity" or "capacity").
+
+    Returns:
+    pd.DataFrame: Updated g_lo DataFrame with relaxed growth constraints.
```

Comment:
Thanks for adding this docstring! Please compare this to e.g. https://github.com/iiasa/message-ix-models/blob/main/message_ix_models/tools/iamc.py#L13-L26 and follow the same structure here. This will ensure that this docstring is rendered properly :)